open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -218,17 +212,33 @@ def augment_audio_signal(signal, sample_freq, augmentation):
   Args:
     signal (np.array): np.array containing raw audio signal.
     sample_freq (float): frames per second.
-    augmentation (dict): dictionary of augmentation parameters. See
-        :func:`get_speech_features_from_file` for specification and example.
+    augmentation (dict, optional): None or dictionary of augmentation parameters.
+        If not None, has to have 'speed_perturbation_ratio',
+        'noise_level_min', or 'noise_level_max' fields, e.g.::
+          augmentation={
+            'speed_perturbation_ratio': 0.2,
+            'noise_level_min': -90,
+            'noise_level_max': -46,
+          }
+        'speed_perturbation_ratio' can either be a list of possible speed
+        perturbation factors or a float. If float, a random value from 
+        U[1-speed_perturbation_ratio, 1+speed_perturbation_ratio].
   Returns:
     np.array: np.array with augmented audio signal.
   """
   signal_float = normalize_signal(signal.astype(np.float32))
 
-  if 'time_stretch_ratio' in augmentation and augmentation['time_stretch_ratio'] > 0:
-    # time stretch (might be slow)
-    stretch_amount = 1.0 + (2.0 * np.random.rand() - 1.0) * \
-                     augmentation['time_stretch_ratio']
+  if 'time_stretch_ratio' in augmentation:
+    print("WARNING: Please update time_stretch_ratio to speed_perturbation_ratio")
```

Comment:
It is better to move in a data layer constructor. We don't want to print the message for every data augmentation call.